CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
       * `.fetchConfigurationFailed` = 7
       * `.missingRedirectURL` = 8
       * `.missingReturnURL` = 9
+      * `.webSessionError` = 10
```

Comment:
This CHANGELOG entry needs to go under `unreleased`. Here, it was added under the 6.0.0-beta3 release, which already went out. The changes you're making in this PR apply to the next unreleased version, which will be tagged upon release.